### ISSUE TITLE
Icons next to podcast names hidden?

### ISSUE THREAD

**Issue by toddmazierski** [CDrummond/lms-material#614]
Hello! Thank you very much for this skin.

In the past, icons appeared next to podcast names. But now they are hidden:

<img width=600 alt="Screenshot of missing podcast icon" src="https://user-images.githubusercontent.com/544541/194726423-590caf53-3498-477c-890a-1f53f6df59f4.png" />

There do appear to be valid images in the `item_loop` array:

```json
{
    "params": {
        "isContextMenu": 1,
        "item_id": "3"
    },
    "icon": "http://www.wfmu.org/podcast_images/td_rss.jpg",
    "infoParams": {
        "name": "Techtonic",
        "url": "https://www.wfmu.org/podcast/TD.xml",
        "image": "http://www.wfmu.org/podcast_images/td_rss.jpg"
    },
    "text": "Techtonic",
    "presetParams": {
        "icon": "http://www.wfmu.org/podcast_images/td_rss.jpg",
        "favorites_type": "link",
        "parser": "Slim::Plugin::Podcast::Parser",
        "favorites_title": "Techtonic",
        "favorites_url": "https://www.wfmu.org/podcast/TD.xml"
    }
}
```

Sorry if this was an intentional change, but I found it useful for quickly scanning the 10+ feeds I have. I'm using LMS v8.2.0 and Material Skin v2.10.5.

Thank you, and please let me know if there are any other details I can provide.

**Comment by CDrummond** [comment]
No, that was not a deliberate change. Will be fixed in next release - 2.10.6

**Comment by toddmazierski** [comment]
Updated this morning. Thank you so much for the fix!
